frontend/app_flowy/packages/flowy_infra/lib/colorscheme/colorscheme.dart

```diff
@@ -67,6 +68,14 @@ abstract class FlowyColorScheme {
   });
 
   factory FlowyColorScheme.builtIn(String themeName, Brightness brightness) {
+    if (themeName == 'Dandelion Community Theme') {
```

Comment:
Please don't hardcode the name. If the translation has been modified then we will find it hard to locate the error.

Reply:
sure